jcl/src/java.base/share/classes/java/lang/invoke/StaticFieldVarHandle.java

```diff
@@ -101,6 +111,7 @@
 	 */
 	StaticFieldVarHandle(Class<?> lookupClass, String fieldName, Class<?> fieldType, Class<?> accessClass) {
 		super(lookupClass, fieldName, fieldType, accessClass, true, COORDINATE_TYPES, populateMHs(fieldType));
+		ensureClassIsInitialized(lookupClass);
```

Comment:
There's no need for the JITHelpers calls here.  A direct call to `_unsafe.ensureClassInitialized(clazz)` is sufficient as this only called on VarHandle creation.

Reply:
Please also add a comment indicating that the change is to match the RI's behaviour, even though its inconsistent with the javadoc.

Reply:
Done.  Thanks.